classes/migration/ConvertSiteWideAboutToRichTextArea.inc.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file classes/migration/ConvertSiteWideAboutToRichTextArea.inc.php
```

Comment:
We don't have a pattern for this yet, but I think issue-specific migration classes should identify the issue numbers in both class names and class filenames, e.g. `I5844ConvertSiteWideAboutToRichTextArea`.

Reply:
Indeed. I've add as a prefix on it. I.e "Issue5844".